packages/matchbox/src/components/Column/Column.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ColumnsContext } from '../Columns/context';
+import styled from 'styled-components';
+import { Box } from '../Box';
+import { gutter } from './styles';
+
+const StyledColumn = styled(Box)`
+  ${gutter}
+`;
+
+const Column = React.forwardRef(function Column(props, ref) {
+  const { width, children } = props;
+  const { space, collapsed } = React.useContext(ColumnsContext);
+
+  return (
+    <StyledColumn
+      width={!width || collapsed ? '100%' : width === 'content' ? 'auto' : width}
```

Comment:
Small nit - I think maybe pull this out to be a little more readable, but up to you